docker/README.md

```diff
@@ -24,27 +24,28 @@ Install [Docker for Desktop](https://www.docker.com/products/docker-desktop) on
+## 4: Prepare your 1Password Account
 
-1. Connect to your remote Docker host from your local machine
-    - Either connect using [docker-machine](https://docs.docker.com/machine/), OR use SSH to access your remote maching and clone this repo.
+Log in to your 1Password account [using this link](https://my.1password.com/scim/setup).  It will take you to a hidden setup page for the SCIM bridge.
+
+Follow the on-screen instructions which will guide you through the following steps:
+
+* Create a Provision Managers group
+* Create and confirm a Provision Manager user
 
-2. In your terminal, use the bash script [./scim-setup.sh](../session/scim-setup.sh) to authenticate your account and generate a `scimsession` file : This script uses a Docker container to run the `op-scim setup` command and writes the scimsession file back to your local machine using a mounted volume. Your bearer token will be printed to the console. **Save your bearer token, as it will be needed to authenticate with your IdP**.
+You can then download the `scimsession` file and save your bearer token.  The `scimsession` file contains the credentials for the new Provision Manager user.  This user will creates, confirms, and suspends users, and creates and manages access to groups.  You should use an email address that is unique and not that of another user.
 
-_The scimsession file is equivalent to your Master Password and Secret Key when combined with the bearer token, therefore they should never be stored in the same place._
+The bearer token and scimsession file combined can be used to sign in to your Provision Manager account. You’ll need to share the bearer token with your identity provider, but it’s important to **never share it with anyone else**. And never share your scimsession file with **anyone at all**.
 
-Example:
-```
-> cd [location of cloned scim-examples folder]
-> ./scim-setup.sh
+You should move your newly created `scimsession` file into the `scim-examples` folder (the root folder of this repository which you cloned earlier).
 
-[interactive script]
-Bearer token: jafewnqrrupcnoiqj0829fe209fnsoudbf02efsdo
 
-> ./docker/deploy.sh
-```
+## 5: Deploy SCIM bridge
+
+1. Connect to your remote Docker host from your local machine
+    - Either connect using [docker-machine](https://docs.docker.com/machine/), OR use SSH to access your remote machine and clone this repo.
 
-3. Once your scimsession file has been created, use the bash script `./docker/deploy.sh` to deploy the SCIM bridge. _Have the domain name indicated by the DNS record created for the SCIM bridge ready_. This script will do the following :
+2. In your terminal, use the bash script [./docker/deploy.sh](deploy.sh) to deploy your SCIM bridge.  If you cloned this repo on your remote machine using SSH, you should copy the `scimsession` file to the `scim-examples` folder on your server. _Have the domain name indicated by the DNS record created for the SCIM bridge ready_. This script will do the following:
```

Comment:
Copying the `scimsession` over the network seems like the option option here.  Should be OK if using `scp` or something?  Again, let me know if this should be changed.